app/app/api/opportunities/[opportunityId]/admins/[adminId]/route.ts

```diff
@@ -0,0 +1,50 @@
+import prisma from "database/db";
+import { NextRequest, NextResponse } from 'next/server';
+
+export const GET = async (request: NextRequest, { params:{opportunityId, adminId} }): Promise<NextResponse> => {  
+
+    const readAdminById = await prisma.opportunityParticipation.findUnique({
+        where: {
+            opportunityParticipationId: {
+                opportunityId: parseInt(opportunityId),
+                userId: adminId
+            },
+            role: "admin"
```

Comment:
Can we use an Enum here to allow for type checking?  We could either define it directly in the schema or in a separate file

Reply:
Yes! Added to the backlog for the weekend :D